src/util/route.js

```diff
@@ -149,3 +150,8 @@ export function handleRouteEntered (route: Route) {
     }
   }
 }
+/** just create a route without any added process */
+export function createPlainRoute (url: string): Route {
```

Comment:
Ideally, this function (and its tests) should be removed since we just need to call `normalizeLocation()` on the url and compare it with `this.current`

Reply:
but how to compare with this current ,this.current is a Route object.I thought if it turn to  a  Route object , the logic can be keep smooth and complete.